<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="letterDice">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Letter dice</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>981</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>photo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="output">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>451</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="numberplate">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Numberplate</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>350</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_4">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>470</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="output_2">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>140</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="output_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1056</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
